commands/ListRoutes/ListRoutes.ts

```diff
@@ -12,18 +12,15 @@ import type { RouterContract } from '@ioc:Adonis/Core/Route'
 import { RoutesPrettyRenderer } from './PrettyRenderer'
 import { RoutesTableRenderer } from './TableRenderer'
 
-export interface SerializedRouter {
-  [domain: string]: SerializedRoute[]
-}
-
 /**
  * Shape of a route serialized by the ListRoute JSON serializer
  */
-export interface SerializedRoute {
```

Comment:
Can we extract the typing from the `@adonisjs/http-server` package.

[adonisjs/http-server@develop/src/Router/index.ts#L271](https://github.com/adonisjs/http-server/blob/develop/src/Router/index.ts#L271)

Reply:
We can. But, I realized that these types are what an internal function within the command returns. Right now they are more or less same, but that internal function should have the freedom on return/decide its own data structure